pkg/types/cose/v0.0.1/entry.go

```diff
@@ -347,3 +347,7 @@ func (v V001Entry) CreateFromArtifactProperties(_ context.Context, props types.A
 
 	return &returnVal, nil
 }
+
+func (v V001Entry) Verifier() (pki.PublicKey, error) {
+	return x509.NewPublicKey(bytes.NewReader(*v.CoseObj.PublicKey))
```

Comment:
why not return `v.keyObj` for this and others?

Reply:
I was concerned that there may be cases where they aren't set, because the use of `keyObj` was not uniform for all types.